backend/src/test/java/de/neuefische/driver/backend/services/MongoUserDetailsServiceTest.java

```diff
@@ -0,0 +1,34 @@
+package de.neuefische.driver.backend.services;
+
+import de.neuefische.driver.backend.models.MongoUser;
+import de.neuefische.driver.backend.repos.MongoUserRepo;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import java.util.Collections;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+class MongoUserDetailsServiceTest {
+    final MongoUserRepo mongoUserRepo = mock(MongoUserRepo.class);
+    final MongoUserDetailsService mongoUserDetailsService = new MongoUserDetailsService(mongoUserRepo);
+
+    @Test
+    void loadUserByUsername_shouldReturnUser() {
+        MongoUser mongoTestUser = new MongoUser("1", "testUser", "test");
+        Mockito.when(mongoUserRepo.findMongoUserByUsername("testUser"))
```

Comment:
Mockito is not required to be written

Reply:
You are right. But the teammates want it, because its easier to understand.